### ISSUE TITLE
Make DelaunayTriangulation clone?

### ISSUE THREAD

**Issue by acshi** [Stoeoef/spade#72]
In some testing code, I found it useful to keep around a less-modified copy of a DelaunayTriangulation, and was surprised it isn't clone! This could be probably be added trivially?

In the mean time I am doing the following as a workaround:
```rust
DelaunayTriangulation::<Point>::bulk_load(
        triangulation
            .vertices()
            .map(|v| v.data().clone())
            .collect(),
```

Alternatively, is there a better workaround suggestion?

**Comment by Stoeoef** [comment]
That's an oversight - `DelaunayTriangulation` should definitely be `Clone`!

It's fixed on my local changeset already... I'm trying to polish them up, then this should be resolved.

**Comment by Stoeoef** [comment]
Fixed with v2.1 (just released) - closing
